docs/cli-guide.rst

```diff
@@ -387,3 +419,54 @@ Anchor use cases
    alignments.
 
    These known timestamps can be converted to anchors.
+
+Silences: inserting pause-like silences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are times where you might want a read-along to pause at a particular
+place for a specific time and resume again after. This can be accomplished by
+inserting silences in your audio stream. You can do it manually by editing your
+audio file ahead of time, but you can also have ``readalongs align`` insert the
+silences for you.
+
+Silence syntax
+^^^^^^^^^^^^^^
+
+Silences are inserted in the audio stream wherever a ``silence`` element is
+found in the XML input.
+**TODO say something about how the silence placement determined.**
```

Comment:
Do we want to merge this with the TODOs? Or did you put these here for me to fill out?

Reply:
@roedoejet Actually, Marc has reported errors in where silences are inserted, especially when there are anchors next to silences. Short answer, yes, I'd like you to fill this in. I'll merge it into dev.cli as is, but it a TODO for you, in conjunction with fixing #89 and Marc's reported silence placement issues.